src/stream/src/task/stream_manager.rs

```diff
@@ -503,12 +505,12 @@ impl LocalStreamManagerCore {
         };
 
         let executor = create_executor(executor_params, self, node, store)?;
-        let executor = Self::wrap_executor_for_debug(
-            executor,
-            actor_id,
-            input_pos,
-            self.streaming_metrics.clone(),
-        )?;
+        // let executor = Self::wrap_executor_for_debug(
+        //     executor,
+        //     actor_id,
+        //     input_pos,
+        //     self.streaming_metrics.clone(),
+        // )?;
```

Comment:
may clean these

Reply:
I‘m gonna migrate debug executors in the next PR (within a single executor🤣).